internal/resources/tanzupackagerepository/constant.go

```diff
@@ -12,4 +12,6 @@ const (
 	NamespaceKey   = "namespace_name"
 	disabledKey    = "disabled"
 	DataSourceRead = "dataSourceRead"
+
+	globalRepoNamespace = "tanzu-package-repo-global"
```

Comment:
Will it always be this?

Reply:
in some other namespace we can not create pkg repository.

Reply:
I mean, the global namespace for uTKG (TKGS on VC8) clusters might be different, please check how to get this global namespace from the backend APIs instead of hardcoding it.

Reply:
Updated PTAL